tests/common.go

```diff
@@ -798,7 +802,7 @@ func ValidatePDB(pdbValue int, allowedDisruptions int, initialNumNodes int, isCl
 		}
 		if initialNumNodes-currentNumNodes > 1 {
 			*isClusterParallelyUpgraded = true
-			
+
```

Comment:
Small nit pick, seems like you have added a space here. Please remove it.

Reply:
removed the whitespace